src/core/core.scale.js

```diff
@@ -310,22 +330,12 @@ var Scale = Element.extend({
 
 		me.afterTickToLabelConversion();
 
+		// IMPORTANT: below this point, we consider that `this.ticks` will NEVER change!
```

Comment:
This comment should be after we modify `this.ticks`. So after next line.

Reply:
done